src/features/points/usePoints.tsx

```diff
@@ -52,3 +58,11 @@ const getMyAvailablePoints = async () => {
     profile.points_checkpointed_at
   );
 };
+
+const nextGiveAvailableAt = (currPoints: number) => {
+  const needed = POINTS_PER_GIVE - (currPoints % POINTS_PER_GIVE);
```

Comment:
Modulus seems strange here

Reply:
for current use, its not necessary since component only shows this if you have < 1 give. But this could be used elsewhere, like "when will I get my 4th Give available" ? In that case, we want to know the time until next give.

Lets say you have 3.2 GIVE available, time to next give should be 1 - .2, not 1 - 3.2. Thus modulus removes the excess, right? @CryptoGraffe